<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_Case</class>
 <widget class="QWidget" name="Add_Case">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Case</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Corpus name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_corpus_name">
       <property name="text">
        <string>name</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_11">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Problem type:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Description:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Keywords:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Original corpus id:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QComboBox" name="cb_problem_type">
             <item>
              <property name="text">
               <string>similarity</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>translation</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_description"/>
           </item>
           <item>
            <widget class="QLineEdit" name="le_summary"/>
           </item>
           <item>
            <widget class="QLineEdit" name="le_original_corpus_id"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Text extension:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Plagiarism type:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Original corpus:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Added by:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QComboBox" name="cb_text_extension">
             <item>
              <property name="text">
               <string>paragraph</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cb_plag_type"/>
           </item>
           <item>
            <widget class="QLineEdit" name="le_original_corpus"/>
           </item>
           <item>
            <widget class="QLineEdit" name="le_generator_name"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Suspicious text:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QTextEdit" name="te_susp_text"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Doc name:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Offset:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Length:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Sentences:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QLineEdit" name="le_susp_doc_name">
               <property name="text">
                <string>susp/</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le_susp_offset">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lb_susp_length">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lb_susp_sentences_count">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Source text:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QTextEdit" name="te_src_text"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>Doc name:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Offset:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Length:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Sentences:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <widget class="QLineEdit" name="le_src_doc_name">
               <property name="text">
                <string>src/</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le_src_offset">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lb_src_length">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lb_src_sentences_count">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_add_case">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cb_problem_type</tabstop>
  <tabstop>cb_text_extension</tabstop>
  <tabstop>le_description</tabstop>
  <tabstop>cb_plag_type</tabstop>
  <tabstop>le_summary</tabstop>
  <tabstop>le_original_corpus</tabstop>
  <tabstop>le_original_corpus_id</tabstop>
  <tabstop>le_generator_name</tabstop>
  <tabstop>te_susp_text</tabstop>
  <tabstop>le_susp_doc_name</tabstop>
  <tabstop>le_susp_offset</tabstop>
  <tabstop>te_src_text</tabstop>
  <tabstop>le_src_doc_name</tabstop>
  <tabstop>le_src_offset</tabstop>
  <tabstop>btn_add_case</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Add_Case</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>539</y>
    </hint>
    <hint type="destinationlabel">
     <x>494</x>
     <y>558</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
